# Generated by Django 2.1.4 on 2019-07-31 12:22

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('artevenue', '0036_auto_20190730_1152'),
    ]

    operations = [
        migrations.CreateModel(
            name='Image_url_error',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('row_id', models.IntegerField(null=True)),
                ('name', models.CharField(max_length=128, null=True)),
                ('err_desc', models.CharField(max_length=2000, null=True)),
                ('error', models.CharField(max_length=2000, null=True)),
                ('created_date', models.DateTimeField(auto_now_add=True)),
                ('updated_date', models.DateTimeField(auto_now=True)),
            ],
        ),
    ]
